content/docs/csidriver/installation/operator/_index.md

```diff
@@ -266,25 +266,64 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
 
 **NOTES:** 
 1. If you are trying to upgrade the CSI driver from an older version, make sure to modify the _configVersion_ field if required.
-2. The parameter "dnsPolicy: ClusterFirstWithHostNet" should be added as follows:
+2. Volume Health Monitoring feature is optional and by default this feature is disabled for drivers when installed via operator.
+   To enable this feature, we will have to modify the below block while upgrading the driver.To get the volume health state `value` 
+   under controller should be set to true as seen below and to get the volume stats `value` under node should be set to true.
+    i. add controller and node section as bellow:
+```yaml
+   controller:
+   envs:
+    - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
+      value: "true"
+      dnsPolicy: ClusterFirstWithHostNet
+      node:
+      envs:
+    - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
+      value: "true"
 ```
-driver:
-    configVersion: v5
-    replicas: 2
-    dnsPolicy: ClusterFirstWithHostNet
-    common:
+   ii. update the sidecar versions:
```

Comment:
Also mention - Add external-health-monitor sidecar to enable health monitor of CSI volumes from Controller plugin

Reply:
tc